invenio_rdm_records/config.py

```diff
@@ -523,3 +523,6 @@ def make_doi(prefix, record):
 
     DATACITE_FORMAT = make_doi
 """
+
+DATACITE_DATACENTER_SYMBOL = ""
```

Comment:
@slint default to empty?

Reply:
Yes, It seems that from the [XML Schema](http://schema.datacite.org/oai/oai-1.0/oai.xsd), the element is required, but there's nothing about the text...